book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] This keeps getting closer, but kindly note that the `add_rental` method is responsible for creating a `Rental` object instead of accepting an already created object so I'd advise you refactor the method to accept 2 arguments `person` & `date` and create a new instance of a rental like this `Rental.new(date, person, self)` in the body of the method instead of pushing to the `rentals` attribute.